build.gradle

```diff
@@ -245,12 +290,19 @@ task javadocZip(type: Zip, dependsOn: javadoc) {
     from javadoc.destinationDir
 }
 
+task license3rdPartyZip(type: Zip) {
```

Comment:
Is zipping (and then unzipping in `preDist`) necessary? Could we not simply copy the `dependency-cache/extracted/fop/license3rdparty` folder into `generated-dist/plugins/${groupName}/license3rdparty` directly? If so, while we're simplifying this let's rename the subfolder to just `generated-dist/plugins/${groupName}/license` and place files with the convention `{software}_license.txt`, e.g. `fop_license.txt`. Seems like a sensible structure used by other Java tools like IntelliJ.

Reply:
Not zip the licenses and put in license folder.